design/components/card.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+
+import { styled } from './stitches.config';
+
+const StyledCard = styled('section', {
+  display: 'inline-block',
+  img: {
+    height: '100%',
+    width: '100%',
+    objectFit: 'cover',
+    transition: `transform .6s ease`,
+  },
+  outline: 'none',
+  position: 'relative',
+
+  'a::after': { position: 'absolute', top: 0, left: 0, right: 0, bottom: 0 },
+
+  '.card-img-container': {
+    height: 324,
+    width: 406,
+    overflow: 'hidden',
+  },
+
+  ':first-child:hover img': {
+    transform: 'scale(1.2)',
+  },
+
+  '.CardContentContainer': {
+    display: 'flex',
+    flexFlow: 'column nowrap',
+    position: 'absolute',
+    top: 264,
+    left: 0,
+    width: 376,
```

Comment:
Still a width :)

Reply:
if i removed the width how can i style the CardContentContainer ??
check this out 
<img width="1260" alt="Screenshot 2020-10-09 at 13 51 47" src="https://user-images.githubusercontent.com/37541400/95580127-18c56200-0a37-11eb-80dd-33278d0fa401.png">
<img width="1260" alt="Screenshot 2020-10-09 at 13 51 55" src="https://user-images.githubusercontent.com/37541400/95580147-21b63380-0a37-11eb-9569-c7f2006ee27c.png">

Reply:
removed